core/src/com/ericc/the/game/components/RenderableComponent.java

```diff
@@ -8,7 +8,8 @@
     public Model model;
     public TextureRegion region;
     public Affine2 transform = new Affine2();
-    public float lightLevel = 0.0f;
+    public float brightness = 0.0f;
+    public boolean visible = false;
```

Comment:
czy to nie jest już za dużo rzeczy w tym jednym komponencie? Jeśli coś ma być invisible to nie lepiej żeby nie miało w ogóle komponentu renderable?